### ISSUE TITLE
Windows Defender scans WSL1 files

### ISSUE THREAD

**Issue by mikuta0407** [microsoft/WSL#7423]
### Windows Build Number

Microsoft Windows [Version 10.0.19043.1165]

### WSL Version

- [ ] WSL 2
- [X] WSL 1

### Kernel Version

Linux version 4.4.0-19041-Microsoft

### Distro Version

Ubuntu 20.04

### Repro Steps

1. In WSL1, create a PHP file with the following contents using an editor such as nano (example: php_system_echotest.php)

```php
<? php

system ("echo test");
```

2. Run `php_system_echotest.php`

3. The console prints the error `Could not open input file: php_system_echotest.php` and Windows Defender detects the threat (see image below).
![image](https://user-images.githubusercontent.com/13357430/133109666-9aa1e964-6ff0-48f5-94d4-3c525664c26b.png)

Image content:

Backdoor: PHP / Dirtelti.MTG
Warning level: Critical
State: Active
Date: 2021/09/13 23:53
Category: Backdoor
Details: This program provides remote access to the installed computer

Affected items: (path to the actual file in the WSL1 file system)

supplement:

-When I try to create the above file using WSL1 and VS Code (Remote - WSL), Windows Defender detects it the moment I save it.

### Expected Behavior

Microsoft has announced that "Do not access WSL1 files from Windows tools", so I thought the files would run undetected.
Or, even if it was detected, I thought that the file path was not the actual file

https://devblogs.microsoft.com/commandline/do-not-change-linux-files-using-windows-apps-and-tools/

### Actual Behavior

Windows Defender scans the WSL1 file system for threat detection.

- The detected file may have the contents remaining but cannot be displayed, or the contents may be rewritten.
- In the image below, php_system_echotest.php (PHP that does `system (" echo test ";)`) has some contents but cannot use xxd and cannot be executed as PHP because it will be detected by Windows Defender.
The contents of php_cleartest.php (PHP the does `system ("clear ";)`) have been changed.

![image](https://user-images.githubusercontent.com/13357430/133109982-bd7ce54f-5a66-4699-bc50-633204247303.png)

**Comment by craigloewen-msft** [comment]
This is expected behavior, in WSL 1 the Linux files are stored as actual Linux files on your machine and all Linux system calls are executed as Windows system calls at the end of the day, both of these are in the domain of Windows Defender.

If you'd like to resolve this, I'd recommend that you upgrade to use WSL 2 instead. Are you able to do so?

**Comment by WSLUser** [comment]
See https://github.com/Microsoft/WSL/issues/1852#issuecomment-371605730 (and probably others for the A/V side of things) 

TLDR: You can add all your pico processes to Windows Defender (Security) to get them exempt.

**Comment by mikuta0407** [comment]
Does direct file access by Windows Defender have a negative impact on WSL1?
If it doesn't affect WSL1, I can use it with confidence.

(I want to use WSL1 not only as a development environment but also as one of the CLI environments, so I would like to continue using WSL1)

**Comment by WSLUser** [comment]
What I can tell you is part of why WSL2 exists is due to the major filesystem performance difference. Even when you exempt your processes, Defender still messes with the performance, presumably due to being treated the same as the rest of the Windows filesystem but likely a bug somewhere so that it doesn't treat the Linux filesystem exactly the same. With WSL2, Defender plays a part but in the same way and has less of an issue. There are threads (probably moved to Discussions) that talk all about filesystem perf. I suggest you check those out.
